### ISSUE TITLE
Contrôle de cohérence sur la taille du groupe si toutes les [taille_totale_groupe] valent [volume_manuel]

### ISSUE THREAD

**Issue by JMPivette** [JMPivette/evavelo#74]
Remonté par Stéphanie:
 

> En première phase de traitement, on peut lancer la vérification de l’intégrité du fichier et un contrôle de cohérence sur la taille du groupe en cas de réponses multiples.
> J’ai une interrogation sur ce dernier point. En effet, EVA-Scan pour les questionnaires multiples identifiés dans les onglets ‘enquete’, va comparer la taille_totale_groupe et le volume_manuel des onglets de comptages manuels.
> Globalement, EVA-Scan détecte bien quand le volume diffère mais pas dans le cas de figure en orange. Je m’interroge sur la raison. Est-ce que je suis passée à côté lors du recettage (je ne trouve pas de trace) ou était-ce voulu (mais avec le recul je m’interroge) ?

> ![image](https://user-images.githubusercontent.com/46813298/145181959-8d6ecbd2-be21-4db8-9235-dccfefa128ec.png)

> Est-ce que tu as une idée sur ce point ?

**Comment by JMPivette** [comment]
Dans la méthode Eva-Vélo, cela correspond au cas n°3 du chapitre **3.1.5.2 Vérification des questionnaires identifiés**

> [...]plusieurs questionnaires pour un groupe, mais chacun renseigne la taille totale du groupe. Le chiffre renseigné correspond au total compté manuellement. **Il n’y a donc pas de correction à faire**, mais pour faciliter les traitements à venir, le volume est reporté à l’identique dans le champ [volume_manuel_valide].

Eva-Scan considère qu'il n'y a pas de correction à apporter et donc pas d'anomalie à détecter.

@VeloTerritoires , faut-il considéré ce cas comme une anomalie ou laisser le fonctionnement actuel?

**Comment by JMPivette** [comment]
Le code à changer se trouve ici:

https://github.com/JMPivette/evavelo/blob/2006d70734e6bf83d2a48f57fc8cf548e22bebcf/R/check_evavelo.R#L295

**Comment by VeloTerritoires** [comment]
Oui, en effet ce n'est pas une anomalie. C'est juste qu'au moment où je t'ai écrit j'ai fait madame plus et je me suis dit ben tient ce serait vachement bien si ces cas étaient signalé pour : faire une vérif et s'assurer qu'il n'y a pas d'erreur quand même + si c'est ok identifier clairement ce cas pour leur appliquer plus facilement le bon traitement derrière ;-).
Mais pas de souci si cela reste comme ça. Faut me freiner sinon je vais te pousser des améliorations à n'en plus finir.
